documentation/modules/ROOT/pages/configuration/topic-auto-create-config.adoc

```diff
@@ -0,0 +1,216 @@
+// Category: debezium-using
+// Type: assembly
+// ModuleID: configuring-debezium-to-auto-create-change-data-capture-topics
+// Title: Configuring {prodname} to use automatically create topics
+[id="cdc-topic-auto-create-config"]
+= Custom Topic Auto-Creation
+
+:toc:
+:toc-placement: macro
+:linkattrs:
+:icons: font
+:source-highlighter: highlight.js
+
+toc::[]
+
+{prodname} automatically creates *internal* topics for offsets, connector status, config
+storage and history topics. The destination topics for the captured tables will be
+automatically created with a default config by the Kafka brokers when
+`auto.create.topics.enable` is enabled.{empty} +
+When topic creation is disabled on the brokers, for example in production environments,
+or when the topics need a different configuration then these topics have to be created
+upfront either automated in a custom deployment process or manually until Kafka Connect 2.6.
+
+Since Kafka 2.6.0 Kafka Connect supports customizable topic auto-creation.
+
+== Kafka Connect
+
+Kafka Connect since Kafka 2.6.0 comes with topic creation enabled:
+
+[source,options="nowrap",shell]
+----
+topic.creation.enable = true
+----
+
+// TODO: how to express this for downstream?
+[NOTE]
+====
+If you don't want to allow automatic topic creation by connectors you can set this value to `false`
+in the Kafka Connect config (_connect-distributed.properties_ file or via environment variable
+_CONNECT_TOPIC_CREATION_ENABLE_ when using https://hub.docker.com/r/debezium/connect[Debezium's container image for Kafka Connect]).
+====
```

Comment:
Can we use that for the downstream docs as well or do we need a different wording here?
What do you suggest? @gunnarmorling  @TovaCohen

Reply:
No, for downstream this should refer to the connector CRD equivalent of this configuration.

Reply:
Do you have an example where we link to a CRD (parallel to the docker hub link)

Reply:
is it `{DockerKafkaConnect}` ?

Reply:
See any connector doc page (under "example configuration").

Reply:
@gunnarmorling This is a CRD for a connector. The config needs to go to the Kafka Connect container env var.

Reply:
see below